docs/attributes-registry/gen-ai.md

```diff
@@ -67,7 +69,9 @@ If none of these options apply, the `gen_ai.system` SHOULD be set to `_OTHER`.
 | `anthropic` | Anthropic | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `aws.bedrock` | AWS Bedrock | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `az.ai.inference` | Azure AI Inference | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `az.ai.openai` | Azure OpenAI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `cohere` | Cohere | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gemini` | Gemini | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `ibm.watsonx.ai` | IBM Watsonx AI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `openai` | OpenAI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `vertex_ai` | Vertex AI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
We can also add perplexity, xai, deepseek and groq to this list

Reply:
yeah I was thinking about xai, just didn't add it yet. happy to.

I didn't know that perplexity deepseek present openai compatible services. Can you link me? Otherwise, I'd prefer this PR to only include OpenAI compatible ones

Reply:
Found links, so will update in a sec

* https://docs.perplexity.ai/api-reference/chat-completions
* https://x.ai/api (note they also say they are compatible with anthropic!)
* https://api-docs.deepseek.com/zh-cn/
* https://console.groq.com/docs/openai